cmd/cas/main.go

```diff
@@ -93,6 +94,6 @@ func main() {
 	wg := sync.WaitGroup{}
 	wg.Add(1)
 	// Poll from Anchor DB and post to the Validate queue to kick-off processing
-	go services.NewRequestPoller(anchorDb, stateDb, validateQueue).Run()
+	go services.NewRequestPoller(anchorDb, stateDb, validateQueue).Run(context.Background())
```

Comment:
This made me remember we need to add graceful shutdown sometime soon (it's already one of the TODOs in the readme).

We should be able to send a shutdown signal and have all services gracefully cleanup and terminate. Right now, we have to kill the task.